composer.json

```diff
@@ -6,11 +6,12 @@
 
     , "require": {
           "php": "^5.5|^7.0"
-        , "illuminate/database": "~5.2.0"
-        , "symfony/dependency-injection": "^2.6|^3.0"
+        , "illuminate/database": "^5.0.0 <5.3"
```

Comment:
The reason for this change is to include 5.0.* and 5.1.* as well. But apparently we can use the `5.0 - 5.2` notation as well (check done [here](https://semver.mwl.be/#?package=illuminate%2Fdatabase&version=5.0%20-%205.2&minimum-stability=stable).